lib/facility_factory.rb

```diff
@@ -0,0 +1,54 @@
+class FacilityFactory
+  def create_facilities(state_raw_data)
+    created_facilities = state_raw_data.map do |facility| 
+      facility_details = format_facility(facility)
+        Facility.new(
+          {
+            name:      facility_details[:name],
+            address:   facility_details[:address],
+            phone:     facility_details[:phone]
+          }
+        )
+      end
+    created_facilities
+  end
```

Comment:
```suggestion
  def create_facilities(state_raw_data)
    state_raw_data.map do |facility|
      facility_details = format_facility(facility)
      Facility.new(
        {
          name:      facility_details[:name],
          address:   facility_details[:address],
          phone:     facility_details[:phone]
        }
      )
    end
  end
```
The indentation is a little off here.

I like that you have your formatting being done as a separate method so that `create_facilities` can be responsible for JUST creating facilities.